### PR TITLE
Created Confirmation Page 

### PR DESCRIPTION
**Pull request by jpcas**

### CODE REVIEW

File: client/src/pages/Confirmation.js [hunk 1]
```diff
@@ -1,22 +1,29 @@
-// import React from 'react';
-// import HeroImage from '../components/HeroImage';
+import React from 'react';
+import ConfirmTitle from '../components/ConfirmTitle';
+import BikeImg from '../components/BikeImg';
+import AwaitBike from '../components/AwaitBike';
+import BikeShopCard from '../components/BikeShopCard';
+import Estimate from '../components/Estimate';
+import DropOff from '../components/DropOff';
 // import InfoCards from '../components/InfoCards';
```
Comment (tracilevine) [inline client/src/pages/Confirmation.js:8]: Remove commented out code lines-8-12

Also line 14 is extra

File: client/src/pages/Confirmation.js [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 // import FeaturedShops from '../components/FeaturedShops';
```
Comment (tracilevine) [inline client/src/pages/Confirmation.js:9]: Remove commented out code.

File: server/routes/secure/reviews.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ const router = require('express').Router({ mergeParams: true });
 const {
   getAllReviews,
   addReview,
```
Comment (tracilevine) [inline server/routes/secure/reviews.js]: Not sure why this is edited

File: client/src/components/BikeShopCard.js [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
   return (
     <div>
-      <Container>
-        <Card>
-          <Card.Img src="holder.js/171x180" alt="Bike Shop Logo" />
-          <Card.Title>Bike Shop Name</Card.Title>
-          <Card.Text>Contact Information</Card.Text>
-        </Card>
-      </Container>
+      <Card className="d-flex flex-row">
```
Comment (tracilevine) [inline client/src/components/BikeShopCard.js:8]: Could this be the same card that we render on the page before and through out the entire project. Could we streamline our components a little?

File: client/src/components/ConfirmTitle.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+
+export default function ConfirmTitle() {
```
Comment (tracilevine) [inline client/src/components/ConfirmTitle.js:4]: Does this need to be it's own component or can it be rendered as a title on the page?

File: client/src/components/DropOff.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+
+export default function DropOff() {
+  return (
+    <Card>
```
Comment (tracilevine) [inline client/src/components/DropOff.js:6]: See comment above? Also Is a card the best layout?

**Review by tracilevine — CHANGES_REQUESTED** [review]

File: client/src/components/BikeShopCard.js (hunk 4 above)
Comment (jpcas) [reply client/src/components/BikeShopCard.js:8]: Yes, it can and would be perfect.

File: client/src/components/ConfirmTitle.js (hunk 5 above)
Comment (jpcas) [reply client/src/components/ConfirmTitle.js:4]: You are right. Done.

**Pull request closed without merging**